react/README.md

```diff
@@ -0,0 +1,257 @@
+# React File Management with Preview - Fully Customized
+
+React file management system built with React, Next.js and TypeScript that allows for single and multiple file uploading with a preview feature. It allows you to select files and preview them, returning an array of selected files. It allows customizing design by overriding the style classes.
+
+---
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [Configuration](#configuration)
+- [Properties and Events](#properties-and-events)
+- [Usage](#usage)
+
+---
+
+## Getting Started
+
+Follow below instructions to configure this package into your project.
+
+### Prerequisites
+
+Before you begin, make sure you have the following software installed:
+
+- [Node.js](https://nodejs.org/) - v20.x
+- React with Next.js
+- sass
+
+### Installation
+
+Using npm:
+
+```
+npm install @canopassoftware/react-file-upload
+```
+
+Using yarn:
+
+```
+yarn add @canopassoftware/react-file-upload
+```
+
+### Configuration
+
+- If you face any `loader` or `typescript` realted issue after started using components from library then add this in your config file.
+
+```js
+transpilePackages: ["@canopassoftware/react-file-upload"],
```

Comment:
I think this is not the best solution. We should fix something in our library. Do not let users fix this issue. Because there are many react libraries in typescript we are not facing issues by using them.

Let's find better solution for this